holmes.go

```diff
@@ -709,7 +707,20 @@ func (h *Holmes) DisableProfileReporter() {
 	atomic.StoreInt32(&h.opts.rptOpts.active, 0)
 }
 
+func (h *Holmes) EnableProfileReporter() {
+	if h.opts.rptOpts.reporter == nil {
```

Comment:
```suggestion
	if opts:=h.opts.GetReporterOpts();opts.reporter== nil {

```
这里不用这个统一的加锁get方法是有什么考量吗

Reply:
忘记了~